openrtx/include/core/state.h

```diff
@@ -81,6 +92,9 @@ enum RtxStatus
     RTX_OFF = 0,
     RTX_RX,
     RTX_TX
+    #ifdef CONFIG_SPECTRUM
+    ,RTX_SPECTRUM
```

Comment:
I'd call this state `RTX_RX_SWEEP` and make it independent from spectrum. This allows you to do a RX sweep also for other applications